app/helpers/resupplies_helper.rb

```diff
@@ -0,0 +1,2 @@
+module ResuppliesHelper
+end
```

Comment:
If, this helper is not necesary, please delete it.

Reply:
Done